### ISSUE TITLE
Accept trailing comma on field extension list.

### ISSUE THREAD

**Issue by Icantjuddle** [protocolbuffers/protobuf#14516]
Right now trailing commas on field extensions are considered to be a missing identifiers.

Re-produce as:
```sh
 protoc \
  <(echo 'syntax = "proto3";\npackage test;\nimport "google/protobuf/descriptor.proto";\nextend google.protobuf.FieldOptions {\n  int32 a = 50000;\n  int32 b = 50001;\n}\nmessage Test {\n  int32 test = 1 [(a) = 1, (b) = 2,];\n}') \ 
  --cpp_out $(mktemp -d) \
  --proto_path /proc/self/fd/
```

(tested on 3.6.1)

Removing the comma at the end of `(b) = 2,` allows it to compile. 

**What language does this apply to?**

Proto3 (presumably the same in proto2)

**Describe the problem you are trying to solve.**

I'd like to be able to supply trailing commas.

**Describe the solution you'd like**

Trailing commas are allowed.

**Comment by haberman** [comment]
What is the use case for this? Is it for easy copy paste when you have one line per option? eg.

```proto
syntax = "proto3";
package test;
import "google/protobuf/descriptor.proto";
extend google.protobuf.FieldOptions {
  int32 a = 50000;
  int32 b = 50001;
}

message Test {
  int32 test = 1 [
      (a) = 1,
      (b) = 2,
  ];
}
```

**Comment by Icantjuddle** [comment]
> What is the use case for this? Is it for easy copy paste when you have one line per option? 

Yes, easy re-ordering of lines would be nice, also seems more consistent.

**Comment by lesomnus** [comment]
How can I make the issue active again? Should I open a new one?

Accepting a trailing comma makes it easier to generate proto files.

**Comment by jguamie** [comment]
We plan to add this to a future Editions release.
